cf/bioinformatics-access-group.yaml

```diff
@@ -0,0 +1,40 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Set up SNS topics for Biomage SCP [managed by github.com/biomage-ltd/iac]
+
+Resources: 
+  StagingAccessGroup: 
```

Comment:
they will also need a role for putting things in Dynamodb - this is for the data ingest script where we add the new experiment to Dynamodb

Reply:
yup, just noticed, fixed